packages/api-kit/tests/e2e/addSafeOperation.test.ts

```diff
@@ -0,0 +1,193 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import { Wallet, ethers } from 'ethers'
+import sinon from 'sinon'
+import sinonChai from 'sinon-chai'
+import { EthAdapter } from '@safe-global/safe-core-sdk-types'
+import SafeApiKit from '@safe-global/api-kit'
+import { Safe4337Pack } from '@safe-global/relay-kit'
+import { generateTransferCallData } from '@safe-global/relay-kit/src/packs/safe-4337/testing-utils/helpers'
+import { RPC_4337_CALLS } from '@safe-global/relay-kit/packs/safe-4337/constants'
+import { EthersAdapter } from 'packages/protocol-kit'
+import { getSafe4337ModuleDeployment } from '@safe-global/safe-modules-deployments'
+import config from '../utils/config'
+import { getServiceClient } from '../utils/setupServiceClient'
+
+chai.use(chaiAsPromised)
+chai.use(sinonChai)
+
+const SIGNER_PK = '0x83a415ca62e11f5fa5567e98450d0f82ae19ff36ef876c10a8d448c788a53676'
+const SAFE_ADDRESS = '0x60C4Ab82D06Fd7dFE9517e17736C2Dcc77443EF0' // 1/1 Safe (v1.4.1) with signer above as owner + 4337 module enabled
```

Comment:
The comment is great! Its not big deal but fo future reference: whenever there is signature logic for a multisig, Id go with a higher threshold else the tests would not cover that logic that much.